### PR TITLE
[PBF-666] Fix for [V4] [Alice][Akamai-hd2] CC with initialTime:1000 i…

### PR DESCRIPTION
**Pull request by sarita-ooyala**
…s not working.

### CODE REVIEW

File: src/akamai/lib/AkamaiHDPlayer.as [hunk 1]
```diff
@@ -263,21 +263,17 @@ package
...
         {
+          if (_initalSeekTime > _streamController.mediaPlayer.duration)
+          {
+            _initalSeekTime = (int) (_streamController.mediaPlayer.duration);
+            dispatchTimeUpdateEvent(_initalSeekTime); 
+          }
+          else if (_initalSeekTime < 0)
+          {
+            return;
+          }
```
Comment (chibzu) [inline src/akamai/lib/AkamaiHDPlayer.as]: This logic is significantly different. Why does a negative initialSeekTime lead to a return here?

File: src/akamai/lib/AkamaiHDPlayer.as [hunk 2]
```diff
@@ -263,21 +263,17 @@ package
     {
       if (event.info.code == "NetStream.Buffer.Full")
       {
-        if (_initialPlay)
```
Comment (chibzu) [inline src/akamai/lib/AkamaiHDPlayer.as]: Is removing this check ok? is NetStream.Buffer.Full only fired once when the video is initially loaded?

File: src/akamai/lib/AkamaiHDPlayer.as (hunk 1 above)
Comment (sarita-ooyala) [inline src/akamai/lib/AkamaiHDPlayer.as]: If initial time is negative, the expected behavior is to start the playback from the beginning(ie; the normal behavior). So nothing is to be done for this condition, hence returning from the function.

File: src/akamai/lib/AkamaiHDPlayer.as (hunk 2 above)
Comment (sarita-ooyala) [inline src/akamai/lib/AkamaiHDPlayer.as]: Checking for _initialPlay is not required at this point.

**Comment by chibzu** [comment]
Alright, looks good.

**Pull request merged**